Jenkinsfile

```diff
@@ -44,15 +43,12 @@ pipeline {
                  postCommentIfPR("Internal build has been started, your results will be available at build completion.", "${GITHUB_USERNAME}", "${GITHUB_REPONAME}", "${GITHUB_TOKEN}")
             }
         }
-        // Use the pomfix tool to validate that bundle dependencies are properly declared
-        stage('Validate Poms') {
+        // Checkout the repository
+        stage('Checkout repo') {
```

Comment:
❓ Is it possible to still validate the poms up front without a new release of pom-lint? I think we did this because we didn't want to go through compilation and tests only to find a missing dependency.

Reply:
Missed this question before. Yes it is. The pom linter is run as part of the validation lifecycle phase so any issues will be caught, even in quick builds.